features/facebookImport/src/components/dialogs/polypolyDialog/polypolyDialog.css

```diff
@@ -25,7 +25,7 @@
 }
 
 .polypoly-dialog .polypoly-dialog-window h2 {
-    font-weight: 600;
+    font-family: "Jost Medium";
```

Comment:
And here's 600, instead of 400 above. Is that because weights were incorrect?

Reply:
Yes, the weights were incorrect because the font didn't fit and the devs tried to match the figma-model with manipulating the weight... I will do a post in the dev-channel soon, how to read figma fonts